### PR TITLE
PLAT-278: pending counters must be updated after init

### PR DESCRIPTION
**Pull request by MikhailZhukov**
**- What I did**
Move updating counters as early as we can
 
**- How I did it**
Refactor code and call updating counters after Init step

**- How to verify it**
Run test

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/conveyor/smachine/state_update_type.go [hunk 1]
```diff
@@ -186,10 +186,10 @@ func (v StateUpdateType) verifyVar(u interface{}) interface{} {
 }
 
 func (v StateUpdateType) ShortLoop(slot *Slot, stateUpdate StateUpdate, loopCount uint32) bool {
-	if v.shortLoop == nil {
-		return false
-	}
-	return v.shortLoop(slot, stateUpdate, loopCount, v.stepDeclaration)
+	// if v.shortLoop == nil {
```
Comment (ivanshibitov) [inline ledger-core/v2/conveyor/smachine/state_update_type.go:189]: Are you sure?

Comment (MikhailZhukov) [reply ledger-core/v2/conveyor/smachine/state_update_type.go:189]: @cyraxred Can you confirm this part?

Comment (ivanshibitov) [reply ledger-core/v2/conveyor/smachine/state_update_type.go:189]: In any case, I think it should be in separate PR

Comment (MikhailZhukov) [reply ledger-core/v2/conveyor/smachine/state_update_type.go:189]: split it in different branch and create PR https://github.com/insolar/assured-ledger/pull/113

File: ledger-core/v2/virtual/integration/pending_counters_test.go [hunk 2]
```diff
@@ -0,0 +1,266 @@
+// Copyright 2020 Insolar Network Ltd.
```
Comment (sexyhamster) [inline ledger-core/v2/virtual/integration/pending_counters_test.go:1]: this test is not integration, it does not require to have running virtual, so it should be placed in execute folder

Comment (MikhailZhukov) [reply ledger-core/v2/virtual/integration/pending_counters_test.go]: Fix it

**Pull request merged**